src/--tests--/searchbar.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import SearchBar from '../SearchBar'
+// import { shallow, mount, render } from 'enzyme'
```

Comment:
Please remove all commented code before making a pull-request. There is no need to keep it around (that is what git is for!) 😄